man/systemctl.xml

```diff
@@ -481,19 +481,15 @@
           <para>When used with <command>enable</command>, overwrite
           any existing conflicting symlinks.</para>
 
-          <para>When used with <command>halt</command>,
-          <command>poweroff</command>, <command>reboot</command> or
-          <command>kexec</command>, execute the selected operation
-          without shutting down all units. However, all processes will
-          be killed forcibly and all file systems are unmounted or
-          remounted read-only. This is hence a drastic but relatively
-          safe option to request an immediate reboot. If
-          <option>--force</option> is specified twice for these
-          operations, they will be executed immediately without
-          terminating any processes or unmounting any file
-          systems. Warning: specifying <option>--force</option> twice
-          with any of these operations might result in data
-          loss.</para>
+          <para>When used with <command>halt</command>, <command>poweroff</command>, <command>reboot</command> or
+          <command>kexec</command>, execute the selected operation without shutting down all units. However, all
```

Comment:
Shouldn't we update the "kexec" section too https://github.com/systemd/systemd/pull/3310/files#diff-7a3aa382affc9b9d042a71a58abdfe7fR1652 ?